openmetadata-ui/src/main/resources/ui/src/components/common/CertificationTag/CertificationTag.tsx

```diff
@@ -29,12 +30,13 @@ const CertificationTag = ({
     const name = getEntityName(certification.tagLabel);
     const actualName = certification.tagLabel.name ?? '';
     const tagSrc = getTagImageSrc(certification.tagLabel.style.iconURL);
+    const tagLink = actualName ? `/tag/Certification.${actualName}` : '';
 
```

Comment:
can you check if we have any utility function to generate tag/certification link ? so in future if path change, then that should also reflect here!